e2e/support/test_support.go

```diff
@@ -1220,6 +1221,18 @@ func CRDs() func() []metav1.APIResource {
 	}
 }
 
+func ConsoleCLIDownload(name string) func() *consoleV1.ConsoleCLIDownload {
+	return func() *consoleV1.ConsoleCLIDownload {
```

Comment:
The group must be registered at https://github.com/apache/camel-k/blob/8523a6d329e53618feb0a35ef85dfe50c1078b3f/e2e/support/test_support.go#L122.

Reply:
hmm, it was working locally. That's strange. Let me fix it..

Reply:
OK, I realise I've been too quick. It's failing in the current OpenShift workflow, not because of the type registration, but because it's still OpenShift 3, that doesn't have the ConsoleCLIDownload resources.

So the test should probably also be skipped depending on the result of:
```go
ok, err := kubernetes.IsAPIResourceInstalled(c, "console.openshift.io/v1", reflect.TypeOf(console.ConsoleCLIDownload{}).Name())
```

Reply:
yes, I started investigating the OCP 3 API once it failed again. I hope it will work this time.